mpt/core/src/main/java/org/apache/james/mpt/protocol/FileProtocolSessionBuilder.java

```diff
@@ -177,6 +178,11 @@ else if (next.startsWith(SESSION_TAG)) {
                     }
                     sessionNumber = Integer.parseInt(number);
                 }
+                else if (next.startsWith(TIMER)) {
+                    TimerCommand timerCommand = TimerCommand.from(next.substring(6, 11));
```

Comment:
Firstly for me 6 and 11 are totally magic number. I think something like:

    int timeCommandIndexStart = next.substring((TIME + " ").length +1;
    TimerCommand.from(next.substring(timeCommandIndexStart, timeCommandIndexStart + TIMER_COMMAND_LENGTH);

But even that, it's not really good imagine latter someone want to introduce a new command that has not the same number of letter than "start", "print", "reset" ? 

I think it will be way better to use regexp something like:

        Pattern timePattern = Pattern.compile("TIMER \([a-z]+\)\s+\([A-Za-z]+\)");
        Matcher m = timePattern.matcher(next);
        timerCommand = m.group(1);
        timerName = m.group(2);